internal/command/repository/create.go

```diff
@@ -23,11 +23,7 @@ func NewCreateCommand() *cli.Command {
 
 			client := c.Context.Value(core.ClientKey).(*client.Client)
 			res, err := client.ResolvePath(c.Context, c.Args().Get(0))
-			if err == nil {
```

Comment:
This was an early exit if the repo already exists.  Otherwise you have to fill the prompts before getting a repo exists error.

Reply:
fixed by calling:

```go
if err != nil && !errors.Is(err, types.ErrRepositoryNotExist) {
	return err
}
```